aspnetcore/data/ef-mvc/intro.md

```diff
@@ -16,20 +16,21 @@ uid: data/ef-mvc/intro
 
 By [Tom Dykstra](https://github.com/tdykstra) and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-The Contoso University sample web application demonstrates how to create ASP.NET Core 1.1 MVC web applications using Entity Framework Core 1.1 and Visual Studio 2017.
+The Contoso University sample web application demonstrates how to create ASP.NET Core 2.0 MVC web applications using Entity Framework Core 2.0 and Visual Studio 2017.
```

Comment:
Entity Framework --> Entity Framework (EF)
- This makes the usage of EF downstream acceptable, even though EF is well-known by most readers.